pyproject.toml

```
@@ -0,0 +1,74 @@
+[build-system]
+requires = [
+    "setuptools>=66.1.0",
+    "setuptools-scm>=8.0",
+    "wheel>=0.38.1",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["any"]
+# Note: The absence of setup.py causes the need for any sub-packages to be
+# listed explicitly. Otherwise, building the wheel archive raises "Warning:
+# Package 'pywbem._vendor.nocaselist' is absent from the `packages`
+# configuration." and complains about an ambiguous configuration.
+packages = [
+    "pywbem",
+    "pywbem._vendor",
+    "pywbem._vendor.nocasedict",
+    "pywbem._vendor.nocaselist",
+    "pywbem_mock"
+]
```

You probably also can try:

```suggestion
packages = {find = {include = ["pywbem*"]}}
```

Ref: https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-namespace-packages

or equivalently:

```toml
[tool.setuptools.packages.find]
include = ["pywbem*"]
```

---

I have changed the "packages" to:
```
[tool.setuptools.packages.find]
include = ["pywbem", "pywbem_mock"]
```
without using the `pywbem*` wildcard, because that seemed too dangerous to me with the flat layout we currently use. I think we'll move to the src layout at some point.

---

I think that you still need the card here https://github.com/pywbem/pywbem/pull/3174#discussion_r1587619278 to avoid errors.